examples/portfolio/src/pages/about.astro

```diff
@@ -35,7 +36,7 @@ import Nav from '../components/Nav/index.jsx';
 					src="https://images.unsplash.com/photo-1581977012607-4091712d36f9?auto=format&fit=crop&w=1400&h=350&q=75"
 				/>
 			</div>
-			<div class="bio wrapper wrapper__readable mt8">
```

Comment:
nit: probably worth in-lining `wrapper__readable` this style to avoid BEM conventions